.gitignore

```diff
@@ -37,7 +37,7 @@ frontend/src/constants/env.ts
 **/locust-scripts/__pycache__/
 **/__debug_bin
 
-frontend/.env
+.env
```

Comment:
We need to ignore frontend/.env file too

Reply:
.env should take care of all the `.env` files be it in any folder